src/org/usfirst/frc/team467/robot/Autonomous/Actions.java

```diff
@@ -18,21 +18,21 @@ public static final Action nothing(){
 		String actionText = "Do Nothing";
 		return new Action(actionText,
 				() -> drive.isStopped(),
-				() -> drive.moveFeet(0));
+				() -> drive.moveForwardFeet(0));
```

Comment:
What happens if you call moveForwardFeet with a negative number? Seems like the name is misleading.